carla_ground_truth_integration/carma_docker.launch.py

```diff
@@ -100,6 +100,9 @@ def generate_launch_description():
         description='List of String: Guidance Control Plugins that will be validated by the Guidance Plugin Validator Node if enabled'
     )
 
+    simulation_mode = LaunchConfiguration('simulation_mode')
+    declare_simulation_mode = DeclareLaunchArgument(name='simulation_mode', default_value = 'False', description = 'True if CARMA Platform is launched with CARLA Simulator')
```

Comment:
This should default to True for the carla_ground_truth_integration configuration.